packages/eslint-plugin-gestalt/src/helpers/eslintASTHelpers.js

```diff
@@ -331,6 +359,25 @@ const getLocalComponentImportName: GetLocalComponentImportNameType = ({
   return (componentNameMatch && componentNameMatch[1]) ?? componentName;
 };
 
+type IsGestaltComponentType = ({|
+  elementNode: GenericNode,
+  gestaltImportNode: GenericNode,
+  componentName: string,
+|}) => boolean;
+
+/** This function ... */
```

Comment:
I'm not sure if I follow these "This function" comments?

Reply:
AGGG sorry, I forgot to write them!

Reply:
DONE